### DISCUSSION TITLE
app crashes

### DISCUSSION THREAD

**Discussion by RSP22** [hbb20/CountryCodePickerProject#495] [Q&A]
my app crashes when i click the country code drop down menu
![1Capture](https://user-images.githubusercontent.com/43504809/123476665-a4f5d480-d61a-11eb-8427-202515db30d7.PNG)
![2Capture](https://user-images.githubusercontent.com/43504809/123476680-a8895b80-d61a-11eb-8e77-73bcce9a4570.PNG)

**Comment by hbb20** [comment]
> my app crashes when i click the country code drop down menu

Can you post the crash report or stack trace?

**Reply by RSP22** [reply]
![2Capture](https://user-images.githubusercontent.com/43504809/123476118-ee91ef80-d619-11eb-8de9-fbaa19b31bf9.PNG)
![1Capture](https://user-images.githubusercontent.com/43504809/123476125-efc31c80-d619-11eb-97a1-360ce4007e99.PNG)

**Reply by hbb20** [reply]
The issues with FastScroller is fixed when we brought that library code right in this library. So please let me know if you still see this issue.
